### ISSUE TITLE
Wrong fexvolume driver directory

### ISSUE THREAD

**Issue by jvanz** [SUSE/doc-caasp#805]
Hi guys! 

I believe we have a problem in the documentation. The path of the directory where the customer should put their Flex Volume driver, seems to be wrong. The [documentation](https://documentation.suse.com/suse-caasp/4.0/single-html/caasp-admin/#_flexvolume_configuration) show the directory as `/usr/libexec/kubernetes/kubelet-plugins/volume/exec/`. However, the spec file path is `%define volume_plugin_dir %{_libexecdir}/kubernetes/kubelet-plugins/volume/exec`. If we check the value of the `%{_libexecdir}` we got:

```bash
rpm --eval "some text printed on %{_libexecdir}"
some text printed on /usr/lib
```

Related to [bsc#1171629](https://bugzilla.suse.com/show_bug.cgi?id=1171629)

**Comment by r0ckarong** [comment]
So what is the correct location then?
This other path seems to come from here: https://github.com/kubernetes/community/blob/master/contributors/devel/sig-storage/flexvolume.md

**Comment by r0ckarong** [comment]
@jvanz Reading this https://github.com/SUSE/avant-garde/issues/1338 it appears to me this is not entirely resolved yet and should be introduced with the next major release. Please confirm.

**Comment by jvanz** [comment]
> If I read this correctly the path should be corrected to: `/usr/lib/kubernetes/kubelet-plugins/volume/exec` ? Does it need the trailing slash or not?

Yeah, for CaaSP v4 that's the right path. I think you can keep the trailing slash

> @jvanz Reading this [SUSE/avant-garde#1338](https://github.com/SUSE/avant-garde/issues/1338) it appears to me this is not entirely resolved yet and should be introduced with the next major release. Please confirm.

That's right. But there is a change in the latest CaaSP v4 already. Which changed the path to `/usr/lib/kubernetes/kubelet-plugins/volume/exec`. In other words, it can change again in CaaSP v5.

**Comment by r0ckarong** [comment]
> That's right. But there is a change in the latest CaaSP v4 already. Which changed the path to `/usr/lib/kubernetes/kubelet-plugins/volume/exec`. In other words, it can change again in CaaSP v5.

Then I will just change the path for v4 and hope that this will come up during v5 development when it happens.
